content/c-sharp/concepts/classes/classes.md

```diff
@@ -0,0 +1,105 @@
+---
+Title: 'Classes'
+Description: 'A class represents a custom data type.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Classes'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'learn-c-sharp-introduction'
+  - 'paths/computer-science'
+  - 'learn-asp-net'
+---
+
+A **class** represents a custom data type. In C#, the class defines the kinds of properties and methods included in a custom type. Generally, a class declaration contains the keyword `class` followed by an identifier (name) of the class.
+
+## Syntax
+
+```pseudo
+// Declaring the class:
+public class ClassName
+{
+  // Example Instance Variables
+  String subName;
+  int exampleInt;
+
+  // Constructor for ClassName class
+  public ClassName(String subName, int exampleInt)
+  {
+    this.subName = subName;
+    this.exampleInt = exampleInt;
+  }
+
+  // Property 1
+  public String GetClassSubName()
+  {
+    return subName;
+  }
+
+  // Property 2
+  public int GetClassExampleInt()
+  {
+    return exampleInt;
+  }
+}
+
+// Calling the class from the main method
+public static void Main(String[] args)
+{
+  // Creating object with Class
+  ClassName Codecademy = new ClassName("LearnCSharp", 2023)
+}
+```
+
+## Codebyte Example
+
+In the example below, a public class is declared as 'Book'. The class and its attributes are first declared. Within the class, it contains a book's name, author, and release date. Within `Main()`, two Book objects are declared: GreatGatsby and HungerGames. Each object then has it's appropriate detail declared within the parameters.
+
+```codebyte/csharp
```

Comment:
These classes should have methods that produce some sort of output that is used after they are called, so that the user sees something happen when they run the codebyte.